Project

```diff
@@ -0,0 +1,124 @@
+#for stars
+import random
+import java.awt.Font as Font
+
+def main():
+
+  #see if the image location can be found
+  try:
+    setMediaPath("C:\Users\mum\Desktop\Falmouth\comp120-tinkering-graphics-1")
+    health = makePicture("Health_Powerup.png")
+    energy = makePicture("Energy_Powerup.png")
+  #if fails request user to locate folder with images
+  except StandardError:
+    showError("Image location could not be found, please locate the folder")
+    setMediaPath()
+ 
+ #request user to choose powerup
+  powerup = requestIntegerInRange("Choose a powerup: health [1] or energy [2]", 1, 2)
+  if powerup == 1:
+    powerupChoice = health
+    powerupName = "Health"
+    powerupText = "Healing"
+  else:
+    powerupChoice = energy
+    powerupName = "Energy"
+    powerupText = "Charging"
+    
+  #getting the picture height helps alignment
+  pic = powerupChoice
+  picHeight=getHeight(pic)
+  picWidth=getWidth(pic)
+  star = makePicture("Star.png")
+  glow = makePicture("Glow.png")
+
+  #set the canvas height and width, black
+  canvasHeight=picHeight+40
+  canvasWidth=picWidth+40
+  canvas=makeEmptyPicture(canvasWidth, canvasHeight, black)
+ 
+  show(canvas) 
+  
+  #make pic centered
+  targetX=(canvasHeight-picHeight)/2
+  targetY=targetX
+
+  #make image move up then down
+  for loop in range (0,4):
+    #number of pixels move
+    maxHeight=4
+
+    #move image up
+    for up in range (0,maxHeight):
+      setAllPixelsToAColor(canvas, black)    
+      copyInto(pic, canvas, targetX, targetY-up)
+      addText(canvas, canvasWidth/4, 14, "This is the " + powerupName + " powerup", white)   
+      
+      #random stars
+      for i in range(0,4):
+        starX = random.randint(0, picWidth)
+        starXEnd = starX+40
+        starY = random.randint(0, picHeight)
+        starYEnd = starY+40
+        copyInto(star, canvas, starX, starY) 
+      
+      #text
+      s = powerupText
+      color = makeColor((random.randint(0,255)), (random.randint(0,255)), (random.randint(0,255)))
+      myFont = makeStyle("Comic Sans", Font.BOLD, 40)
+      xpos = picWidth/4
+      ypos = picHeight-40
+      addTextWithStyle(canvas, xpos, ypos, s, myFont, color)
+                
+      repaint(canvas)
+      
+      #make image lighter
+      for px in getPixels(pic):     
+        r1=getRed(px)*1.05
+        g1=getGreen(px)*1.05
+        b1=getBlue(px)*1.05
+
+        newColor=makeColor(r1,g1,b1)
+        setColor(px,newColor)   
+               
+    #move image down
+    for down in range (0,maxHeight):
+      setAllPixelsToAColor(canvas, black)
+      addText(canvas, canvasWidth/4, 14, "This is the " + powerupName + " powerup", white) 
+      copyInto(pic, canvas, targetX, targetY-maxHeight+down)
+      
+      #random stars
+      for i in range(0,4):
```

Comment:
This code is repeated. Perhaps it could be made into a function so that it can be reused.